backend/graphql/types/lessonType.ts

```diff
@@ -0,0 +1,32 @@
+import { gql } from "apollo-server-express";
+
+// TO DO: Fix content typing
+
+const lessonType = gql`
+  type LessonDTO {
+    id: ID!
+    course: ID!
+    title: String
+    description: String
+    image: String
+    content: [ID!]
```

Comment:
content should be an array of Content objects not IDs (see Slack for that discussion). I don't think there's something like the `Object` type in GraphQL, but if you can find one, feel free to use it. Otherwise, just define a dummy `Content` GraphQL type and put it in here. It can just be an empty type for now.

Reply:
+1 to this